packages/flutter/lib/src/widgets/app.dart

```diff
@@ -91,6 +93,9 @@ class WidgetsApp extends StatefulWidget {
   /// Checkerboards raster cache images.
   final bool checkerboardRasterCacheImages;
 
+  /// Checkerboards layers rendered to offscreen bitmaps.
+  final bool checkerboardOffscreenLayers;
```

Comment:
Please include all the information from the render layer here (also on `checkerboardRasterCacheImages` while we're at it).